tours/src/main/kotlin/edu/artic/tours/AllToursViewModel.kt

```diff
@@ -67,9 +67,9 @@ class AllToursViewModel @Inject constructor(
 }
 
 /**
- * This class is fundamentally the same as `WelcomeTourCellViewModel` in the :welcome module.
+ * ViewModel responsible for building each item in the tour summary list.
  */
-class AllToursCellViewModel(
+class TourCellViewModel(
```

Comment:
let's move these into their own files now that they are being used in both places

Reply:
Good idea.